### PR TITLE
Add posibility to add DAO members in bulk

### PR DESCRIPTION
**Pull request by maxmagic-dev**

### CODE REVIEW

File: astro_2.0/components/ProposalCardRenderer/components/ProposalCard/ProposalCard.module.scss [hunk 1]
```diff
@@ -26,6 +26,7 @@
       'proposalType proposalType countdown'
       'proposer proposer .'
       'description description description'
+      'group group group'
```
Comment (AlexeyPonomarenko) [inline astro_2.0/components/ProposalCardRenderer/components/ProposalCard/ProposalCard.module.scss:30]: Should we mention the "group" somewhere in lines 12-17?

File: astro_2.0/features/CreateProposal/components/UpdateGroupContent/UpdateGroupContent.tsx [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+  if (getDataFromContext && formContext) {
+    const { watch } = formContext;
+
+    groupsList = watch('groups');
+  } else {
+    groupsList = groups;
+  }
+
+  const [activeGroupSlug, setActiveGroupSlug] = useState(groupsList[0].slug);
```
Comment (AlexeyPonomarenko) [inline astro_2.0/features/CreateProposal/components/UpdateGroupContent/UpdateGroupContent.tsx:35]: Is there a chance that groupsList might be empty?

File: astro_2.0/features/CreateProposal/components/UpdateGroupContent/UpdateGroupContent.tsx [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+  const activeGroup =
+    groupsList.find(group => group.slug === activeGroupSlug) || groupsList[0];
+
+  return (
+    <div className={styles.root}>
+      <div className={styles.list}>
+        <h4 className={styles.listTitle}>Groups</h4>
```
Comment (AlexeyPonomarenko) [inline astro_2.0/features/CreateProposal/components/UpdateGroupContent/UpdateGroupContent.tsx:43]: Can I ask you to mark files that require localisation with the
```
// TODO requires localisation
```
comment at the top of the file, please?

File: features/groups/helpers.ts [hunk 4]
```diff
@@ -50,7 +50,7 @@ type ContractRole = {
   // eslint-disable-next-line camelcase,@typescript-eslint/ban-types
   vote_policy: Record<string, VotePolicyRequest> | {};
 };
-
+/*
```
Comment (AlexeyPonomarenko) [inline features/groups/helpers.ts:53]: Can we just drop this code or it might be useful?

**Review by dmytro-almighty — APPROVED** [review]

**Pull request merged**